include/sound/sof/dai-amd.h

```diff
@@ -18,4 +18,12 @@ struct sof_ipc_dai_acp_params {
 	uint32_t fsync_rate;    /* FSYNC frequency in Hz */
 	uint32_t tdm_slots;
 } __packed;
+
+/* ACPDMIC Configuration Request - SOF_IPC_DAI_AMD_CONFIG */
+struct sof_ipc_dai_acpdmic_params {
+
```

Comment:
Remove the blank line?

You don't need `struct sof_ipc_hdr hdr;` in here?

Reply:
hdr filed is not used in our firmware code hence not needed

Reply:
lets remove the blank line @ajitkupandey and in the commit message you mention "send the DAI CONFIG IPC" but there's no code to back it up. So, maybe better not to mention it as it is not done during topology parsing

Reply:
Done